configs/templates/_ec2.txt

```diff
@@ -89,65 +99,67 @@ LOGIN = $EC2_LOGIN
 SSH_KEY_NAME = $EC2_SSH_KEY_NAME
 
 [VM_TEMPLATES : EC2_CLOUDCONFIG ]
+ACMEAIR = size:t2.medium, imageid1: cb_acmeair
+APACHE = size:t2.medium, imageid1:cb_wrk
+BONNIE = size:t2.medium, imageid1:cb_bonnie
+BTEST = size:t2.medium, imageid1:cb_btest
+CASSANDRA = size:t2.medium, imageid1:cb_ycsb
 CHECK = size:m3.medium, imageid1:to_replace
 CLIENT_IBM_DAYTRADER = size:t2.micro, imageid1:cb_daytrader
 CLIENT_OPEN_DAYTRADER = size:t2.micro, imageid1:cb_open_daytrader
-DRIVER_DAYTRADER = size:t2.medium, imageid1:cb_daytrader
 CLIENT_TRADELITE = size:t2.medium, imageid1:cb_tradelite
-DRIVER_TRADELITE = size:t2.medium, imageid1:cb_tradelite
-MYSQL = size:t2.medium, lb_size:t2.large, imageid1:cb_open_daytrader
-GERONIMO = size:t2.medium, imageid1:cb_open_daytrader
+CLIENT_WINDOWS = size:t2.medium, imageid1:cb_windows
+CN_HPC = size:t2.medium, imageid1:cb_hpcc
+COREMARK = size:t2.micro, imageid1:cb_coremark
 DB2 = size:t2.medium, lb_size:t2.large, imageid1:cb_daytrader
-WAS = size:t2.medium, imageid1:cb_daytrader
+DDGEN = size:t2.medium, imageid1:cb_hpcc
+DRIVER_COREMARK = size:t2.micro, imageid1:cb_coremark
```

Comment:
Same concern here for Amazon. Basically we kind of "spam" the console during initial CB startup with errors that are not really errors.

If the user hasn't defined any images yet (or hasn't used the workload builder), then `CHECK` really is the only image that needs to be here. 

I've been wanting to remove these for a long time....